src/lib/spinel/radio_spinel.hpp

```diff
@@ -858,7 +858,7 @@ template <typename InterfaceType, typename ProcessContextType> class RadioSpinel
 private:
     enum
     {
-        kMaxSpinelFrame        = SpinelInterface::kMaxFrameSize,
+        kMaxSpinelFrame        = OPENTHREAD_CONFIG_PLATFORM_RADIO_MAX_SPINEL_FRAME,
```

Comment:
Thanks @chapongatien 

I agree that changing this would be good. I feel there are two issues:
- The `SpinelInterface::kMaxFrameSize` constant name itself may not be a good as it is really the total rx buffer size (which can contain many frames and not just one). 
- The fact that this constant is used in `radio_spinel.hpp` here as `kMaxSpinelFrame` is therefore a (big) overestimate.

We have a definition in `spinel.h` itself for max frame size, we can consider using that here and define `kMaxSpinelFrame = SPINEL_FRAME_MAX_SIZE` in `radio_spinel.h`.
```c++
/**
 * @def SPINEL_FRAME_MAX_SIZE
 *
 *  The maximum size of SPINEL frame.
 *
 */
#define SPINEL_FRAME_MAX_SIZE 1300
```

Thoughts?

Reply:
Thanks for the proposal. I agree we can use the SPINEL_FRAME_MAX_SIZE value from spinel.h instead of defining a new one. I will update the PR with this new change.